lxd/daemon.go

```diff
@@ -632,88 +658,89 @@ func startDaemon() (*Daemon, error) {
 		NotFound.Render(w)
 	})
 
```

Comment:
So instead of re-indenting half the function, can you not let the normal socket initialization code run and simply not spawn the listeners at the end?